### ISSUE TITLE
False positive: 'scrollable-region-focusable' rule fails lists without focusable elements on page load

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2822]
```scrollable-region-focusable``` rule fails the scrollable lists which do not contain any focusable elements on initial page load. But, when interacting with keyboard, keyboard focus is added to elements inside the list and the complete content of the list is accessible.

Motivation: All the scrollable content is reachable/available when interacting with keyboard, 

Demo link: https://developer.microsoft.com/en-us/fluentui#/controls/web/list

![image](https://user-images.githubusercontent.com/22068434/109364816-7cb0c180-7844-11eb-847f-b55654b6e5f8.png)

<pre><code>axe-core version: 4.1.2-canary.3cdbcd7
axe version: axe-coconut 4.7.3</code></pre>

**Comment by WilcoFiers** [comment]
Yikes, that is a scenario we did not foresee. We'll have to think about it... this is a tricky one.
